gateways/class-wc-ebanx-safetypay-gateway.php

```diff
@@ -57,13 +61,29 @@ public function __construct() {
 	 * @throws Exception Throws missing param message.
 	 */
 	public function is_available() {
-		$country    = $this->get_transaction_address( 'country' );
+		$country               = $this->get_transaction_address('country');
+		$parent_available      = parent::is_available();
+		$country_iso           = Country::fromIso($country);
+		$available_for_country = $this->ebanx_gateway->isAvailableForCountry( $country_iso );
+
+		if (!empty($country_iso)) {
+			if ($country_iso === Country::PERU && !$this->enabled_in_peru) {
```

Comment:
This line says: `is not available BECAUSE is not enabled` and the 78 says it is or is not available. A method could be available but not enabled. This will tell us why the method is not appearing.